src/services/storage/drivers/relational/index.ts

```diff
@@ -91,11 +110,38 @@ export class RelationalStorage implements StorageDriver {
 			if (!url) throw NotFoundError()
 			return storedUrl
 		}
-		public async save(url: string): Promise<StoredUrl> {
-			const [storedUrl] = await this.storage.db.table<StoredUrl>('urls').insert({ url }).returning('*')
+
+		public async save(urlBody: UrlRequestData): Promise<StoredUrl> {
+			const url = urlBody.url
+			const urlInfo: Partial<UrlWithInformation> = {
+				ip: urlBody.ip,
+				urlVisitCount: 0,
+				infoVisitCount: 0,
+				lastUse: new Date().toISOString(),
+			} as UrlWithInformation
 
 			if (!url) throw NotFoundError()
+			const [storedUrl] = await this.storage.db.table<StoredUrl>('urls').insert({ url }).returning('*')
+			await this.storage.db
```

Comment:
Could there be any way we can unite these two calls into one? If not, it's fine.

Reply:
its a bit problematic since the the second call value depend on the first response...

Reply:
Ok